sdk/kiosk/src/utils.ts

```diff
@@ -70,7 +71,11 @@ export async function getKioskObject(
 }
 
 // helper to extract kiosk data from dynamic fields.
-export function extractKioskData(data: DynamicFieldInfo[]): KioskData {
+export function extractKioskData(
+  data: DynamicFieldInfo[],
+  listings: KioskListing[],
+  lockedItemIds: string[],
+): KioskData {
   return data.reduce<KioskData>(
     (acc: KioskData, val: DynamicFieldInfo) => {
       const type = getTypeWithoutPackageAddress(val.name.type);
```

Comment:
We don't need that, all Kiosk fields are `0x2` prefixed.

Reply:
We needed it, because of the 0x0000....002 thing. Are we certain this is fixed now?